Hey @ppknap, I was bitten by your pupils again, probably sorting could calm them down:

```
--- FAIL: TestMountBook (0.00s)
    --- PASS: TestMountBook/paths_from_mount_A (0.00s)
    --- FAIL: TestMountBook/shared_remote_path (0.00s)
    	mount_test.go:86: want IDs = [334fdc48-c175-4f1b-ba43-634b93848e40 59f65441-c3d7-4723-b35e-c9a0cdfca4f0], got [59f65441-c3d7-4723-b35e-c9a0cdfca4f0 334fdc48-c175-4f1b-ba43-634b93848e40]
```

--- a/go/src/glide.lock
+++ b/go/src/glide.lock
@@ -559,7 +559,7 @@ imports:
   - helpers
   - services
 - name: github.com/koding/kite
-  version: 077dc6df5fcfc0419e399f68445dc633f8cfc853
+  version: 4e1bc06aecf5e401757e07aef8eb8fa723e6ab7d
   subpackages:
   - config
   - dnode

